org.polymap.p4/src/org/polymap/p4/imports/ShapeImportPanel.java

```diff
@@ -167,15 +168,15 @@ protected void updateListAndFAB() {
         fileList.refresh();
         Optional<File> shp = files.stream().filter( f -> f.getName().toLowerCase().endsWith( ".shp" ) ).findAny();
         if (shp.isPresent()) {
-            Button fab = tk.createFab();
-            fab.setToolTipText( "Import this uploaded Shapefile" );
             fab.addSelectionListener( new SelectionAdapter() {
                 @Override
                 public void widgetSelected( SelectionEvent ev ) {
                     importFiles( shp.get() );
                 }
             } );
+            fab.setVisible( true );
         }
+        UIUtils.deactivateCallback( "upload" );
```

Comment:
Wenn es hier nicht steht, wird der Callback erst in org.polymap.rap.updownload.upload.Upload.dispose() weggeräumt, d.h. erst wenn das Import-Panel geschlossen wird. Das reicht wahrscheinlich. Andererseits wird bei jemand widgetSelected Event auf dem Upload-Button der Callback neu geadded.